packages/react-liveness/jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
+  // ignore coverage for top level "export", version and style files
+  coveragePathIgnorePatterns: ['<rootDir>/src/(index|styles|version).(ts|tsx)'],
+  coverageThreshold: {
+    global: {
+      branches: 65,
+      functions: 77,
+      lines: 77,
+      statements: 78,
```

Comment:
Can we add a follow up task to get these all to a minimum of 80?

Reply:
https://app.asana.com/0/1202067976849710/1204357382742891